### DISCUSSION TITLE
I am getting an error in the application

### DISCUSSION THREAD

**Discussion by RookieS193** [community/community#127463] [Programming Help] [answered]
### Select Topic Area

Product Feedback

### Body

SyntaxError: Non-UTF-8 code starting with '\xfc' in file C:\Users\ferid\OneDrive\Masaüstü\Blank-Grabber-main\Blank Grabber\env\Scripts\pyinstaller-script.py on line 2, but no encoding declared; see https://python.org/dev/peps/pep-0263/ for details

**Comment by davevad93** [comment, marked as answer]
Hi @RookieS193 ,

That is due to the encoding of the file which isn't `utf-8`, so you have to declare it by putting one of these two lines at the top of your code. 

`# coding: iso-8859-1 -*-` or `# -*- encoding: utf-8 -*-`

That should fix it.
